js/modules/k6/grpc/client.go

```diff
@@ -144,6 +144,10 @@ func (c *Client) Invoke(
 	params map[string]interface{},
 ) (*grpcext.Response, error) {
 	state := c.vu.State()
+
+	if req == nil {
```

Comment:
Could you please move this check closer to the `b, err := req.ToObject(c.vu.Runtime()).MarshalJSON()` that way, a validation request logic will be located at the same place, which makes it more readable :relaxed:

Reply:
yes of course, thank you for the feedback - let me get this over to you shortly